src/PlasmaFormulary.jl

```diff
@@ -16,10 +16,6 @@ temperature(eot) = uconvert(u"K", eot, Thermal())
 temperature(T::Unitful.Temperature) = uconvert(u"K", T)
 
 # Physical Constants (SI)
-gravitational_constant = 6.67430e-11 * u"m^3 / s^2 / kg"
```

Comment:
Why remove the gravitational constant? Even though none of the current formulas use it, it may be useful to have available to users.

Reply:
I think since many constants are already provided in the `Unitful` package, it feels redundant to include .
Or if we want to include, we should just use something like `gravitational_constant=Unitful.G`